modules/appeals_api/app/swagger/appeals_api/v2/responses_contestable_issues.json

```diff
@@ -0,0 +1,151 @@
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "id": { "type": ["integer", "string"], "nullable": true, "example": null },
+                            "approxDecisionDate": {
+                              "type": "string",
+                              "nullable": true,
+                              "format": "date",
+                              "example": "2006-11-27"
+                            }
+                          }
+                        }
+                      },
+                      "ratingIssueSubjectText": {
+                        "type": "string",
+                        "nullable": true,
+                        "description": "Short description of RatingIssue",
+                        "example": "Hypertension"
+                      },
+                      "ratingIssuePercentNumber": {
+                        "type": "string",
+                        "nullable": true,
+                        "description": "Numerical rating for RatingIssue",
+                        "example": "10"
+                      },
+                      "isRating": { "type": "boolean", "description": "Whether or not this is a RatingIssue", "example": true }}}}}}}}}}},
+  "404": {
+    "description": "Veteran not found",
+    "content": {
+      "application/vnd.api+json": {
+        "schema": {
+          "type": "object",
+          "properties": { "errors": { "$ref": "#/components/schemas/errorWithTitleAndDetail" } }},
+        "examples": {
+          "bad file number or SSN": {
+            "value": {
+              "errors": [{ "status": 404, "code": "veteran_not_found", "title": "Veteran Not Found" }]}}}}}},
+  "422": {
+    "description": "Bad receipt date",
+    "content": {
+      "application/vnd.api+json": {
+        "schema": {
+          "type": "object",
+          "properties": { "errors": { "$ref": "#/components/schemas/errorWithTitleAndDetail" } }},
+        "examples": {
+          "before AMA": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_receipt_date", "title": "Invalid Receipt Date", "detail": "\"2019-02-18\" is before AMA Activation Date (2019-02-19)." }]}},
+          "future receipt date": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_receipt_date", "title": "Invalid Receipt Date", "detail": "\"2020-07-02\" is in the future (today: 2020-07-01; time zone: (GMT-05:00) America/New_York)." }]}},
+          "unparsable date": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_receipt_date", "title": "Invalid Receipt Date", "detail": "\"Widdershins\" is not a valid date." }]}},
+          "invalid veteran ssn": {
+            "value": {
+              "errors": [{ "status": 422, "code": "invalid_veteran_ssn", "title": "Invalid Veteran SSN", "detail": "SSN regex: /^\\d{9}$/)." }]}}}}}},
```

Comment:
I don't believe this is how this error will turn out, as the DetailedSchemaError should catch this as a common schema pattern mismatch error. Is there a different validator we've written that results in this special SSN-specific code & detail?